gdal/ogr/ogrsf_frmts/geomedia/ogrgeomedialayer.cpp

```diff
@@ -165,6 +165,7 @@ CPLErr OGRGeomediaLayer::BuildFeatureDefn( const char *pszLayerName,
             break;
 
           case SQL_C_TIMESTAMP:
+          case SQL_TYPE_TIMESTAMP:
```

Comment:
This constant is SQL data type, but shouldn't this be constant for the [C data type](https://docs.microsoft.com/en-us/sql/odbc/reference/appendixes/c-data-types) `SQL_C_TYPE_TIMESTAMP`?
Numeric value of both can be equal though.

Reply:
Thanks, fixed